.cirrus.yml

```diff
@@ -50,6 +50,11 @@ task:
         - ./script/check_publish.sh
     - name: format
       format_script: ./script/incremental_build.sh format --fail-on-change
+      license_script:
+        - cd script/tool
+        - pub get
+        - cd ../..
+        - dart script/tool/lib/src/main.dart license-check
```

Comment:
Is this an incremental command, or does it check the whole repo on every run?

Reply:
The latter. Originally I wrote it with the intention of having two modes, complete with a flag for doing only changed files and a marker to myself to implement it once everything else was done. Then I discovered that the version that checks everything takes <5 seconds on my machine, so it seemed like extra work and complexity for nothing 🤷

Reply:
> I discovered that the version that checks everything takes <5 seconds on my machine

D A M N